downstream/modules/platform/proc-aap-controller-backup.adoc

```diff
@@ -18,8 +18,7 @@ Use this procedure to back up a deployment of the controller, including jobs, in
 . Select the *Automation Controller Backup* tab.
 . Click btn:[Create AutomationControllerBackup].
 . Enter a *Name* for the backup.
-. Enter the *Deployment name* of the deployed {PlatformNameShort} instance being backed up.
-For example, if your {ControllerName} must be backed up and the deployment name is `aap-controller`, enter 'aap-controller' in the *Deployment name* field.
+. In the *Deployment name* field, enter the name of the AutomationController custom resource object of the deployed {PlatformNameShort} instance being backed up. This name was created when you xref:aap-create_controller[created your AutomationController object].
```

Comment:
```suggestion
. In the *Name* field, enter the name of the AutomationController custom resource object of the deployed {PlatformNameShort} instance being backed up. This name was created when you xref:aap-create_controller[created your AutomationController object].
```

It's not Deployment name in the form either:

<img width="1148" alt="Monosnap Create AnsibleAutomationPlatform · Red Hat OpenShift 2024-11-06 10-55-21" src="https://github.com/user-attachments/assets/fbc808ba-1029-43a7-8814-e8e112e2c22e">